README.md

```diff
@@ -131,12 +131,13 @@ headless = Headless.new(:video => { :frame_rate => 12, :codec => 'libx264' })
 Available options:
 
 * :codec - codec to be used by ffmpeg
-* :frame_rate    - frame rate of video capture
-* :provider      - ffmpeg provider - either :libav (default) or :ffmpeg
-* :pid_file_path - path to ffmpeg pid file, default: "/tmp/.headless_ffmpeg_#{@display}.pid"
-* :tmp_file_path - path to tmp video file,  default: "/tmp/.headless_ffmpeg_#{@display}.mov"
-* :log_file_path - ffmpeg log file,         default: "/dev/null"
-* :extra         - array of extra ffmpeg options, default: [] 
+* :frame_rate      - frame rate of video capture
+* :provider        - ffmpeg provider - either :libav (default) or :ffmpeg
+* :provider_binary - Explicit path to avconv or ffmpeg.  Only required when the binary cannot be discovered on the system $PATH.
```

Comment:
Do you think `provider_binary` is a good name? Do you think `binary_file_path` or `provider_binary_file_path` be better?